lib/alephant/sequencer/sequence_table.rb

```diff
@@ -66,10 +66,24 @@ def update_sequence_id(ident, value, last_seen_check = nil)
             })
           end
 
-          logger.info("SequenceTable#update_sequence_id: with new value #{value} for #{ident} success!")
+          logger.info(
+            "event"  => "SequenceIdUpdated",
+            "id"     => ident,
+            "value"  => value,
+            "method" => "#{self.class}#update_sequence_id"
+          )
         rescue AWS::DynamoDB::Errors::ConditionalCheckFailedException
           logger.metric "SequencerFailedConditionalChecks"
-          logger.warn("SequenceTable#update_sequence_id: (Value to put: #{value}, existing: #{current_sequence}) #{ident} outdated!")
+          logger.error(
+            "event"                => "DynamoConditionalCheckFailed",
+            "newSequenceValue"     => value,
```

Comment:
@Integralist Let's be more specific and refer to it as `newSequenceId` (to indicate specifically that it's an ID).  This would apply in a couple other places too